src/utils/utils.ts

```diff
@@ -155,7 +162,13 @@ export function primitiveTypeToCairo(
     return `u${bits}` as CairoPrimitiveIntType;
   }
 
-  return 'felt';
+  console.log(
+    warning('Warning:'),
+    `Unknown translation for primitive type ${typeString}.`,
+    'Defaulting to felt252',
+  );
+  return `felt252`;
+  // throw new Error(`Unknown translation for cairo type: ${typeString}`);
```

Comment:
Yes, I added the warning because I thought you were working on that, and as a temporay patch. Also I don't think untyped literals will actually give errors on the transpiled contract.

Reply:
By using `felt252` is the same as ignoring it. If it is not `felt252` since we do not annotate it, it will be inferred correctly by the transpiler.

I am not saying that I am willing to defend this solution on the future, only that for now it seems a safe compromise

Reply:
Oh, I did have a patch for other things for that function, but as you pointed out they were better handled in other places, so I don't really do anything with it.

Reply:
So do you agree on leaving it as a warning for now, and if it becomes a reason for failure (or there are no longer other pressing matter), fix it?

Reply:
I don't see any benefit of not raising an Error here: if we don't know what it is, it might not do what we want. I might be missing something. Do however you judge is the best.

Reply:
I undestand your point and I agree with it. If we raise an error then we must make a parser for literals like "int_const 12345" and assign them a type. Also since we have not yet the full scope of which typestring is arriving to this functions, the warning serves as way to probe/learn the types that come into here. 

The purpose of this PR is not to introduce a fully implemented feature but a ground base. It will get very big if all were to be fixed here

Reply:
I see